opentrons-ai-client/src/molecules/ChatDisplay/index.tsx

```diff
@@ -4,30 +4,19 @@ import { Trans, useTranslation } from 'react-i18next'
 import Markdown from 'react-markdown'
 import { useAtom } from 'jotai'
 import { delay } from 'lodash'
```

Comment:
not introduced in this PR but small nit. this way, you don't import the whole `lodash` package and instead only the specific fn you're using.

```suggestion
import delay from 'lodash/delay'
```

Reply:
Changed. Thanks!